db/models/product_review.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require('sequelize');
+const db = require('APP/db');
+
+const Product_Review = db.define('product_review', {
+  body: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  stars: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+    defaultValue: 0,
+    validate: {
+      is: '/\d(?![0-5]\S)/g',
```

Comment:
if you want a single digit between 1 and five you can just do: /^[0-5]/
This is saying the beginning of a line should contain one digit between 0 and 5.
I don't know if regex will validate properly here though as the data type is an integer.